consensus/core/src/dag_state.rs

```diff
@@ -970,6 +1037,21 @@ impl DagState {
     }
 }
 
+struct BlockInfo {
```

Comment:
`BlockInfo` seems odd because it contains the block itself? Unless you plan on adding more to this struct maybe `BlockWithCommitStatus` or `BlockWithStatus`?

Reply:
I am aware that @mwtian will most probably re-use this struct with additional info for Fast Path, that's why kept the `BlockInfo`